site/assets/adopters/clusterpedia.yaml

```diff
@@ -5,3 +5,5 @@ contacts:
 links:
   - name: Clusterpedia E2E Test
     url: https://github.com/clusterpedia-io/clusterpedia/tree/main/test
+  - name: Multi-NIC CNI E2E Test
```

Comment:
We should create a new file in this directory.

Reply:
Sorry. Made new changes [here](https://github.com/kubernetes-sigs/kwok/compare/5638f8766822e17aa66879a2cda06c5e9515eb76..1294d8bc75b82e694b8eba1a640b6c9f824bcc1d).